src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceFileLogger.cs

```diff
@@ -87,7 +89,11 @@ public virtual async Task ProcessLogQueue(object state)
             while (!_cancellationTokenSource.IsCancellationRequested)
             {
                 await InternalProcessLogQueue();
-                await Task.Delay(TimeSpan.FromSeconds(FlushFrequencySeconds), _cancellationTokenSource.Token).ContinueWith(task => { });
+                await Task.Delay(TimeSpan.FromSeconds(StartingFlushFrequencySeconds), _cancellationTokenSource.Token).ContinueWith(task => { });
+                if ( StartingFlushFrequencySeconds < MaxFlushFrequencySeconds)
+                {
+                    StartingFlushFrequencySeconds++;
```

Comment:
Nit: because you're updating this, I would call it CurrentFlushFrequencySeconds, not "starting"